rewrite-maven/src/test/java/org/openrewrite/maven/tree/MavenRepositoryTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.maven.tree;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MavenRepositoryTest {
+    @Test
+    @DisplayName("verify URI of local Maven repository")
+    void expectedLocalMavenRepository() {
+        String expectedPath = "file://" + System.getProperty("user.home") + "/.m2/repository";
```

Comment:
@timtebeek and @fabapp2, this test also needs to be updated accordingly if it's kept. Otherwise, the tests will fail on Windows.

Reply:
I'm fine to remove the test class; no need to test `Paths.get`.